network.go

```diff
@@ -156,6 +159,29 @@ func (endpoint *VirtualEndpoint) SetProperties(properties types.Result) {
 	endpoint.Properties = properties
 }
 
+func networkLogger() *logrus.Entry {
+	return virtLog.WithField("subsystem", "network")
+}
+
+// Attach for virtual endpoint bridges the network pair and adds the
+// tap interface of the network pair to the hypervisor.
+func (endpoint *VirtualEndpoint) Attach(h hypervisor) error {
```

Comment:
@amshinde unrelated to this PR. I will rewrite this to have different virtual endpoints with their own Attach and detach methods.